[2022-08-30 00:30:25,592] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: csv_dag.link_dag scheduled__2022-08-29T00:00:00+00:00 [queued]>
[2022-08-30 00:30:25,610] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: csv_dag.link_dag scheduled__2022-08-29T00:00:00+00:00 [queued]>
[2022-08-30 00:30:25,611] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-30 00:30:25,611] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-08-30 00:30:25,612] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-30 00:30:25,640] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): link_dag> on 2022-08-29 00:00:00+00:00
[2022-08-30 00:30:25,649] {standard_task_runner.py:52} INFO - Started process 25411 to run task
[2022-08-30 00:30:25,655] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'csv_dag', 'link_dag', 'scheduled__2022-08-29T00:00:00+00:00', '--job-id', '281', '--raw', '--subdir', 'DAGS_FOLDER/testDag.py', '--cfg-path', '/tmp/tmpo_qsmqie', '--error-file', '/tmp/tmplkso7qvb']
[2022-08-30 00:30:25,657] {standard_task_runner.py:80} INFO - Job 281: Subtask link_dag
[2022-08-30 00:30:25,762] {task_command.py:371} INFO - Running <TaskInstance: csv_dag.link_dag scheduled__2022-08-29T00:00:00+00:00 [running]> on host 36e1df4dd6ce
[2022-08-30 00:30:25,863] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=csv_dag
AIRFLOW_CTX_TASK_ID=link_dag
AIRFLOW_CTX_EXECUTION_DATE=2022-08-29T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-08-29T00:00:00+00:00
[2022-08-30 00:31:13,312] {python.py:173} INFO - Done. Returned value was: None
[2022-08-30 00:31:13,326] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=csv_dag, task_id=link_dag, execution_date=20220829T000000, start_date=20220830T003025, end_date=20220830T003113
[2022-08-30 00:31:13,356] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-30 00:31:13,392] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
